dedupe/variables/base.py

```diff
@@ -58,21 +58,16 @@ def all_subclasses(
 
 
 class DerivedType(Variable):
-    type = "Derived"
```

Comment:
why did you remove this?

Reply:
I might be wrong but my understanding is:

This would only be used if someone passes in a variable definition like `{"type": "Derived"}`, and then we would create an instance of DerivedType, but that doesn't make sense because DerivedType is an abstract base class and shouldn't ever be created directly.